ProjectTraning/Border.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ProjectTraning
+{
+    public class Border: Draw
```

Comment:
Since all your entities know about console and all are able to draw itself, this approach does not follow SRP principle.

Reply:
-1 to Single responsibility principle.